app/views/layouts/_user_menu.html.haml

```diff
@@ -21,7 +21,10 @@
     = link_to(destroy_user_session_path, :method=>'delete') do
       %span.fa.fa-minus
       Sign out
-- if can? :manage, Conference
+- if current_user.is_admin || (current_user.has_any_role? :admin, { :name => :organizer, :resource => :any },
```

Comment:
Please check your syntax: http://breakthebit.org/post/8453341914/ruby-19-and-the-new-hash-syntax

We discussed extensively about this implementation and I thought we agreed it is best to use cancancan methods to check for abilities?